plugins/search/providers/elastic_search/lib/api/types/data/KalturaMediaEsearchExportToCsvJobData.php

```diff
@@ -12,9 +12,15 @@ class KalturaMediaEsearchExportToCsvJobData extends KalturaExportCsvJobData
 	 * @var KalturaESearchEntryParams
 	 */
 	public $searchParams;
+	/**
+	 * options
+	 * @var KalturaExportToCsvOptionsType
+	 */
+	public $options;
```

Comment:
@TalDubovKaltura we discussed this - the $options needs to be an array allowing to pass several different options, rather than just one.

Reply:
You're right, we already spoke about that. I haven't reached it yet due to QA issues I had and this change will almost start the same process again so I will talk Oleg about that, to understand how much time can we have more.